src/scmdata/run.py

```diff
@@ -988,6 +988,58 @@ def _meta_column(self, col) -> pd.Series:
         out = self._meta.get_level_values(col)
         return pd.Series(out, name=col, index=self._df.columns)
 
+    def set_meta(self, dimension: str, value: Any, **filter_kwargs: Any):
+        """
+        Update metadata of timeseries
+
+        Optionally, a subset of timeseries may be modified through the use of
```

Comment:
```suggestion
        Optionally, a subset of metadata may be modified through the use of
```

This may be just a terminology thing, but it seems to make more sense to talk about metadata here rather than timeseries as I associate timeseries with the numerical values, and we're not changing those.

Reply:
Fair. I tend to think of a timeseries as the numerical values + the associated metadata